src/main/java/Car.java

```diff
@@ -0,0 +1,39 @@
+import java.util.Random;
+
+public class Car {
+    private static final int RANDOM_LIMIT_NUM = 4;
+
+    public int makeRandom() {
+        Random random = new Random();
+        return random.nextInt(10);
```

Comment:
```suggestion
        return new Random().nextInt(10);
```
저는 이렇게 사용하는게 좀 더 직관적인거 같아서 좋아합니다😅